### PR TITLE
Fix - Non LDAP user login

### PR DESCRIPTION
**Pull request by shilpaexaltcg**
- Fixed login error for non LDAP user
- Display logged in user name for Github user in top menubar

### CODE REVIEW

File: src/views/Main/index.jsx [hunk 1]
```diff
@@ -297,12 +289,6 @@ class MainContainer extends Component {
             </Suspense>
             {doneLoading === false && <Spinner loading /> }
           </div>
-          <BottomNavigation
```
Comment (sylvestre) [inline src/views/Main/index.jsx]: why are you removing this?

File: src/utils/getAllReportees.js [hunk 2]
```diff
@@ -31,15 +31,17 @@ const buildOrgChartData = (people) => {
...
-};
-
 const findReportees = (completeOrg, email) => {
   let allReportees = {};
-  allReportees[email] = completeOrg[email];
-  const { reportees } = completeOrg[email];
+
+  // if non-LDAP user, replace user email by the last email in list
+  const allEmails = Object.keys(completeOrg);
+  const checkedEmail = (email in completeOrg) ? email : allEmails[allEmails.length - 1];
```
Comment (calixteman) [inline src/utils/getAllReportees.js]: I don't understand why you're getting the last email in the list.
Could you be more explicit in your comment? Explain what is the case you're handling here, what are the conditions to be here...

File: src/views/Main/index.jsx (hunk 1 above)
Comment (shilpaexaltcg) [reply src/views/Main/index.jsx]: Did not intend to remove this, adding it back.

**Comment by sylvestre** [comment]
@shilpaexaltcg could you please preserve the initial pr from @TrangEmily in https://github.com/mozilla/bugzilla-dashboard/pull/80/files and add a commit on top of that?
it is important to preserve their work and show what you changed compared to the their work.
thanks

**Comment by sylvestre** [comment]
Could you please also do https://github.com/mozilla/bugzilla-dashboard/pull/110#issuecomment-499507653 ? thanks

**Comment by shilpaexaltcg** [comment]
Sure @sylvestre I'll do it.

**Pull request closed without merging**